scripts/release/validate_release.py

```diff
@@ -14,13 +14,13 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+from absl import flags
 import collections
 import fnmatch
-import gflags
 import json
 import logging
 import os
-import prettytable
+from prettytable import PrettyTable
```

Comment:
Why this change?

Reply:
python3 package name changed to PrettyTable.